gcc/rust/privacy/rust-privacy-reporter.cc

```diff
@@ -48,11 +49,19 @@ PrivacyReporter::check_for_privacy_violation (const NodeId &use_id,
 {
   NodeId ref_node_id = UNKNOWN_NODEID;
 
+  rust_debug ("ARTHUR: use_id = %d", use_id);
+
   // FIXME: Don't assert here - we might be dealing with a type
   if (!resolver.lookup_resolved_name (use_id, &ref_node_id))
     resolver.lookup_resolved_type (use_id, &ref_node_id);
 
-  rust_assert (ref_node_id != UNKNOWN_NODEID);
+  // FIXME: Assert here. For now, we return since this causes issues when
+  // checking inferred types (#1260)
+  // rust_assert (ref_node_id != UNKNOWN_NODEID);
+  if (ref_node_id == UNKNOWN_NODEID)
+    return;
+
+  rust_debug ("ARTHUR: resolved_id = %d", ref_node_id);
```

Comment:
Maybe check this debug message to check_for_privacy_violation: hirid %d, ref_node_id